src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -107,17 +101,13 @@ public class BlobFileStore : IFileStore
             BlockBlobClient blob = GetInstanceBlockBlob(versionedInstanceIdentifier);
 
             Stream stream = null;
-            StorageTransferOptions storageTransferOptions = new StorageTransferOptions
-            {
-                MaximumConcurrency = _blobDataStoreConfiguration.RequestOptions.DownloadMaximumConcurrency,
-            };
+            var blobOpenReadOptions = new BlobOpenReadOptions(allowModifications: false);
```

Comment:
Should we make this a `private` `static` member?

Reply:
I shied away from that because one of the properties you can set is `Position`. That lead me to believe that we would likely vary it in the future to make interesting use of this method.